### PR TITLE
allow multiple properties per defaulter, call from __setattr__

### PR DESCRIPTION
**Pull request by azaroth42**

### CODE REVIEW

**Comment by azaroth42** [comment]
The flake8 failure is the relative imports. But I have no idea what it wants. Running isort doesn't solve it.

File: iiif_prezi3/helpers/auto_fields.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import uuid
 
 from ..config.config import Config, register_config
-from ..skeleton import Annotation, Canvas, Collection, Manifest, Range
+from ..skeleton import Annotation, Canvas, Collection, Manifest, Range, KeyValueString
```
[the remark this answers is not in the document]
Comment (p-galligan) [reply iiif_prezi3/helpers/auto_fields.py:4]: That should work for fixing flake8.

File: iiif_prezi3/helpers/auto_fields.py [hunk 2]
```diff
@@ -71,31 +97,41 @@ def generate_id(self, what, auto_type=None):
...
+            # Currently only generate from scratch
+            # Future versions might allow passing the slug to be turned into a URI
+            return None
+        else:
+            return self.generate_id(what)
 
 
-class AutoLabelLangConfig(Config):
+class AutoLangConfig(Config):
     def __init__(self, auto_lang="en"):
```
Comment (glenrobson) [inline iiif_prezi3/helpers/auto_fields.py:110]: Default to none

**Comment by azaroth42** [comment]
Per discussion on 2022-05-20, change default to `none` from `en`

eg

```
>>> from iiif_prezi3 import Manifest
>>> m = Manifest(label="string")
>>> m.json()
'{"id": "http://example.org/iiif/1", "type": "Manifest", "label": {"none": ["string"]}, "metadata": null, "summary": null, "requiredStatement": null, "rendering": null, "service": null, "services": null, "viewingDirection": null, "rights": null, "start": null, "logo": null, "navDate": null, "provider": null, "seeAlso": null, "thumbnail": null, "homepage": null, "behavior": null, "partOf": null, "items": null, "structures": null, "annotations": null}'

>>> m.label = "a different string"
>>> m.json()
'{"id": "http://example.org/iiif/1", "type": "Manifest", "label": {"none": ["a different string"]}, "metadata": null, "summary": null, "requiredStatement": null, "rendering": null, "service": null, "services": null, "viewingDirection": null, "rights": null, "start": null, "logo": null, "navDate": null, "provider": null, "seeAlso": null, "thumbnail": null, "homepage": null, "behavior": null, "partOf": null, "items": null, "structures": null, "annotations": null}'
```

**Pull request merged**